protocol/daemons/liquidation/client/client.go

```diff
@@ -2,28 +2,43 @@ package client
 
 import (
 	"context"
+	"github.com/dydxprotocol/v4-chain/protocol/daemons/server/types"
+	timelib "github.com/dydxprotocol/v4-chain/protocol/lib/time"
 	"time"
 
-	"github.com/dydxprotocol/v4-chain/protocol/lib"
-
-	gometrics "github.com/armon/go-metrics"
 	"github.com/cometbft/cometbft/libs/log"
-	"github.com/cosmos/cosmos-sdk/telemetry"
-	"github.com/cosmos/cosmos-sdk/types/query"
 	appflags "github.com/dydxprotocol/v4-chain/protocol/app/flags"
 	"github.com/dydxprotocol/v4-chain/protocol/daemons/flags"
 	"github.com/dydxprotocol/v4-chain/protocol/daemons/liquidation/api"
 	daemontypes "github.com/dydxprotocol/v4-chain/protocol/daemons/types"
-	"github.com/dydxprotocol/v4-chain/protocol/lib/metrics"
 	clobtypes "github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
 	satypes "github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
 )
 
+// Client implements a daemon service client that periodically queries a gRPC server for all liquidatable subaccounts
+// and reports them back to the protocol via the daemon server.
+type Client struct {
+	// Compose HealthCheckableImpl to implement the HealthCheckable interface.
```

Comment:
update comment to be in sync w [this](https://github.com/dydxprotocol/v4-chain/pull/715/files#diff-d0cb3538fda4a5cbae8939680e81fdfeca343874dd974cc6295d8c8f4ebe507dR34)?